sources/light_client.move

```diff
@@ -270,6 +295,28 @@ public fun calc_next_required_difficulty(c: &LightClient, last_block: &LightBloc
     return new_bits
 }
 
+public(package) fun find_prev_difficulty_testnet(c: &LightClient, start_node: &LightBlock): u32 {
+    // let mut iter_block_hash = start_node.header().block_hash();
+    // let mut iter_block = c.get_light_block_by_hash(iter_block_hash);
```

Comment:
we can remove the commented code above, right?

Reply:
Yes 👍🏼